beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/BeekeeperPathSchedulerApiary.java

```diff
@@ -76,4 +71,18 @@ public static void stop() {
   public static MeterRegistry meterRegistry() {
     return meterRegistry;
   }
+
+  @VisibleForTesting
+  public static void resetStaticContext() {
```

Comment:
I'm surprised this works with nothing having to be closed...

Reply:
😅 yeah, not super happy with how this is setup but it works. I think I was able to get away with it as on first setup of the tests for these, every static variable is already mocked so we're effectively just blowing away if the mock is set or not for those contexts.